client/signup/jetpack-connect/controller.js

```diff
@@ -39,37 +39,27 @@ export default {
 		next();
 	},
 
-	connect( context ) {
-		context.store.dispatch( setSection( 'jetpackConnect', {
-			hasSidebar: false
-		} ) );
```

Comment:
(An alternative would be to use our [`setSection` middleware](https://github.com/Automattic/wp-calypso/pull/4820/files#diff-b5556c136203f1d3cf7ae888ba50316aR33) (e.g. at https://github.com/Automattic/wp-calypso/pull/4820/files#diff-74eaea272a7a7bf4a8633efae6c02123R35).)